simplify sample usage (#623)
JetBrains/skiko: DEVELOPMENT.md

## Building JVM bindings

* Set JAVA_HOME to location of JDK, at least version 11

> * If you have **Apple silicon CPU (M1)**, better to use **X86** JDK architecture to compile JS target.

Reviewer: Doesn't it work otherwise? What happens otherwise?

Author: For me, on Arm64 JDK, sometimes I have error with yarn or webpack. (Like failed to install yarn)
I am also trying to use `kotlin.js.webpack.major.version=4`

With change JDK to X86 JDK, all works fine.
And for now, also Arm64 JDK works fine. Maybe it's a cache problem. I decided to mention JDK change as one of the possible solutions.

Reviewer: I see. Thanks!

Author: I changed a little bit this sentence to better understanding.

### samples/SkiaAwtSample
```bash
./gradlew :skiko:publishAllSkikoJvmRuntimeToMavenLocal &&\
./gradlew :SkiaAwtSample:run -Pskiko.version=0.0.0-SNAPSHOT
```
 - skiko.fps.longFrames.show=true
```bash
./gradlew :skiko:publishAllSkikoJvmRuntimeToMavenLocal &&\
./gradlew :SkiaAwtSample:run -Dskiko.fps.longFrames.show=true -Pskiko.version=0.0.0-SNAPSHOT
```
 - skiko.renderApi=SOFTWARE
```bash
./gradlew :skiko:publishAllSkikoJvmRuntimeToMavenLocal &&\
./gradlew :SkiaAwtSample:run -Dskiko.renderApi=SOFTWARE -Pskiko.version=0.0.0-SNAPSHOT
```
 - skiko.vsync.enabled=false
```bash
./gradlew :skiko:publishAllSkikoJvmRuntimeToMavenLocal &&\
./gradlew :SkiaAwtSample:run -Dskiko.vsync.enabled=false -Pskiko.version=0.0.0-SNAPSHOT
```
 - skiko.transparency=true
```bash
./gradlew :skiko:publishAllSkikoJvmRuntimeToMavenLocal &&\
./gradlew :SkiaAwtSample:run -Dskiko.transparency=true -Pskiko.version=0.0.0-SNAPSHOT
```
 - skiko.swing.interop=true
```bash
./gradlew :skiko:publishAllSkikoJvmRuntimeToMavenLocal &&\
./gradlew :SkiaAwtSample:run -Dskiko.swing.interop=true -Pskiko.version=0.0.0-SNAPSHOT
```

### samples/SkiaJsSample with mavenLocal

> If page in browser will be empty, try to reload it.

Reviewer 2: ```suggestion
If the page in the browser is empty, try to reload it.
```

Anyway, its seems like a bug, and better to not mention it in Readme? Because if the page is empty, reloading it usually is the first thing to try.

Author: Done

Author: Yeah, I think we can skip this comment. It's obviously.

```bash
./gradlew publishToMavenLocal -p skiko -Pskiko.native.enabled=true -Pskiko.wasm.enabled=true &&\
./gradlew jsBrowserDevelopmentRun -p samples/SkiaJsSample -Pskiko.version=0.0.0-SNAPSHOT
```

### samples/SkiaJsSample with skikoCompositeBuild=true

> It means: includeBuild("skiko") with dependency substitution. It compiles faster.

Reviewer 2: ```suggestion
When you add `-DskikoCompositeBuild=true`, Skiko from [sources](https://github.com/JetBrains/skiko/tree/master/skiko) is used (without this property the maven artifact is used). Every time you change sources, they are automatically applied when you run `jsBrowserDevelopmentRun`.
```

`includeBuild` usually doesn't tell what this property does, even if people are familiar with the Compose Gradle build feature. Also, it is used not to build project faster, but to use skiko from sources, not as prepublished maven artifact.

It also means, that we don't need to call `publishSkikoWasmRuntimePublicationToMavenLocal` and add  `-Pskiko.version=0.0.0-SNAPSHOT`.

But see my other suggestion, if we do that way (use Composite build by default), we don't need this section at all.

Author: We still need to call `publishSkikoWasmRuntimePublicationToMavenLocal`.
It's a different jar, packaged by tricky script.
